interop/cats/src/main/scala/caliban/interop/fs2/Fs2Interop.scala

```diff
@@ -0,0 +1,38 @@
+package caliban.interop.fs2
+
+import caliban.interop.cats.FromEffect
```

Comment:
Can you organize imports everywhere?

Reply:
Actually, I did organize the imports. I use VSCode+Metals and this is how their function "Organize imports" works.

Would you prefer to have them organized in another style? Like folded one or something? I could either try to find out if it is possible to configure VSCode that way or even re-organize them by hand, if it'd make sense.

Reply:
Ah yeah I use the IntelliJ style which is group items together and use `_` if there are more than 5 of them. You can see how it is in other files. Now sure if Metals lets you do that easily 😄

Reply:
@ghostdogpr, seems I figured out how to configure the "organize imports" action in Metals. And I managed to set it to an IntelliJ-like style, check it out please.

Also I added a `.scalafix.conf` to the root of the repo which does the trick. So that if someone else also would be using Metals, they could get the IntelliJ style in there without any hustle. But if you don't like that change – let me know please, I am up to roll it back.

Reply:
I think that's ok 👍